stable/mongodb-replicaset/README.md

```diff
@@ -31,56 +31,57 @@ helm install --name my-release stable/mongodb-replicaset
 
 The following table lists the configurable parameters of the mongodb chart and their default values.
 
-| Parameter                           | Description                                                               | Default                                             |
```

Comment:
Why is every line of this table changed but you only added one new parameter? It makes it hard to see the changes.

Reply:
Because original version 3.6.0 README table was not well format. I realign it to look nicer.